kinetic/distribution.yaml

```diff
@@ -16832,6 +16832,21 @@ repositories:
       url: https://github.com/warthog-cpr/warthog_simulator.git
       version: kinetic-devel
     status: maintained
+  watson_ins:
+    doc:
+      type: git
+      url: https://github.com/AutonomousVehicleLaboratory/watson_ins.git
```

Comment:
The source here has a license specified in the package.xml (GPLv3), but no COPYING file at the top-level or headers on the source files.  Please see https://www.gnu.org/licenses/gpl-3.0.html on how to do that.